tests/test.cpp

```diff
@@ -306,21 +306,25 @@ void AccessFlatBufferTest(const uint8_t *flatbuf, size_t length,
   }
   TEST_EQ(n, inv_vec.size());
 
-  n = 0;
-  for (auto it = inventory->rbegin(); it != inventory->rend(); ++it, ++n) {
-    auto indx = inventory->rend() - it - 1;
-    TEST_EQ(*it, inv_vec.at(indx));  // Use bounds-check.
-    TEST_EQ(*it, inv_data[indx]);
+  {
```

Comment:
Additional `{}` code block is unnecessary.

Reply:
That was to limit the scope of the variable inv_vec_indx to the test case. Not strictly necessary, true.